src/phone-number-input/phone-number-input.tsx

```diff
@@ -66,8 +60,14 @@ export const PhoneNumberInput = ({
         }
     };
 
+    const handleBlur = () => {
+        if (onBlur) {
+            onBlur();
+        }
+    };
+
     const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-        const currentValue = event.target.value;
+        const currentValue = event.target.value.replace(/[^0-9]/g, "");
```

Comment:
This is a better way than handling key down. I previously failed to check actions like Copying, pasting, selecting, tabbing. The keydown method is too restrictive